tmt/steps/provision/testcloud.py

```diff
@@ -215,26 +241,17 @@ def options(cls, how=None):
 
     def default(self, option, default=None):
         """ Return default data for given option """
-        defaults = {
-            'user': 'root',
-            'memory': 2048,
-            'disk': 10,
-            'image': 'fedora',
-            'connection': 'session',
-            'arch': platform.machine()
-            }
-        if option in defaults:
-            return defaults[option]
-        return default
+        return getattr(TestcloudGuestData(), option, default)
 
     def wake(self, keys=None, data=None):
         """ Wake up the plugin, process data, apply options """
         super().wake(keys=keys, data=data)
 
         # Convert memory and disk to integers
-        for key in ['memory', 'disk']:
-            if isinstance(self.get(key), str):
-                self.data[key] = int(self.data[key])
+        # TODO: can they ever *not* be integers at this point?
```

Comment:
@happz seems the only leftover item from this MR, it looks clean to me

Reply:
I'd leave this for later, it's not important enough.